SIAB/spillage/api.py

```
@@ -37,153 +37,270 @@ def _coef_restart(fcoef: str):
+    """
+    call = _coef_opt_jy if jy else _coef_opt_pw
+    option = {"maxiter": siab_settings.get("max_steps", 2000),
+              "disp": True, "ftol": 0, "gtol": 1e-6, 'maxcor': 20}
+    nthreads = siab_settings.get("nthreads", 4)
+    return call(rcut, siab_settings, folders, option, nthreads)
+
 def _peel(coef, nzeta_lvl_tot):
     from copy import deepcopy
     coef_lvl = [deepcopy(coef)]
     for nzeta_lvl in reversed(nzeta_lvl_tot):
         coef_lvl.append([[coef_lvl[-1][l].pop(0) for _ in range(nzeta)] for l,nzeta in enumerate(nzeta_lvl)])
```

Ruff

Ambiguous variable name: `l`

(E741)